proto/viam/app/v1/app.proto

```diff
@@ -371,8 +371,10 @@ message DeleteOrganizationMemberResponse {}
 message LocationOrganization {
   // Organization ID the location is shared with.
   string organization_id = 1;
+  // Organiztion name the location is shared with.
+  string organization_name = 2;
   // Whether the organization is the primary owner or not.
-  bool primary = 2;
+  bool primary = 3;
```

Comment:
oops this GitHub Action failure detection is legitimate! We can't reorder the fields in a proto. We'll have to make `organization_name` set to 3. See the [official docs](https://protobuf.dev/programming-guides/proto3/#assigning) for more on assigning field numbers.